app/views/admin/categories/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<% provide(:title, 'Category')%>
+<p></p>
+<h1>Create category</h1>
+
+<div class="row justify-content-center">
+  <div class="col-md-6">
+        <%= render 'form' %>
```

Comment:
@shaniajubelag Should be 4-spaced, not 6

Reply:
![Selection_022](https://user-images.githubusercontent.com/35232348/61602978-5ee5a880-ac6e-11e9-87b7-5351eb5639a1.png)
It was already 4 in my VS Code.

Reply:
@inahsanjuan I will adjust a line